spread.yaml

```diff
@@ -614,19 +636,6 @@ suites:
             "$TESTSLIB"/prepare-restore.sh --restore-suite-each
         restore: |
             "$TESTSLIB"/prepare-restore.sh --restore-suite
-    # Essential tests run inside the autopkgtest environment on each
-    # platform. Autopkgtest can be very slow so we cannot run all
-    # tests there without running into timeouts.
-    tests/smoke/:
-        summary: Essenial system level tests for snapd
-        prepare: |
-            "$TESTSLIB"/prepare-restore.sh --prepare-suite
-        prepare-each: |
-            "$TESTSLIB"/prepare-restore.sh --prepare-suite-each
-        restore-each: |
-            "$TESTSLIB"/prepare-restore.sh --restore-suite-each
-        restore: |
```

Comment:
I think the restore is needed to make sure next suites are running as it is supposed to run